src/app/helpers/helperMethods.ts

```diff
@@ -1,28 +1,34 @@
+import * as path from "path";
+import * as request from "request";
+import * as unzip from "unzipper";
 const fs = require('fs');
+const zipFile = 'project.zip';
 
 export namespace helperMethods {
-    export function deleteFolderRecursively(projectFolder: string) 
-    {
-        if(fs.existsSync(projectFolder))
-        {
-            fs.readdirSync(projectFolder).forEach(function(file,index){ 
-            var curPath = projectFolder + "/" + file; 
-            
-            if(fs.lstatSync(curPath).isDirectory())
-            {
-                deleteFolderRecursively(curPath);
-            }
-            else
+    function deleteFolderRecursively(projectFolder: string) {
+        try {
+            if(fs.existsSync(projectFolder))
             {
-                fs.unlinkSync(curPath);
+                fs.readdirSync(projectFolder).forEach(function(file,index){ 
+                var curPath = projectFolder + "/" + file; 
+                
+                if(fs.lstatSync(curPath).isDirectory())
+                {
+                    deleteFolderRecursively(curPath);
+                }
+                else
+                {
+                    fs.unlinkSync(curPath);
+                }
+            }); 
+            fs.rmdirSync(projectFolder); 
             }
-        }); 
-        fs.rmdirSync(projectFolder); 
+        } catch (err) {
```

Comment:
Recommend having the Error thrown include a message about the operation being performed.
```
throw new Error(`Unable to delete folder "${projectFolder}".\n${err}`);
```

Reply:
addressed locally